frontend/src/Metamaps/Listeners.js

```diff
@@ -31,6 +31,11 @@ const Listeners = {
         case 27: // if esc key is pressed
           JIT.escKeyHandler()
           break
+        case 38: // if UP key is pressed
+          if(e.ctrlKey && e.shiftKey){
```

Comment:
Can you change this to

(e.ctrlKey || e.metaKey) && e.shiftKey

I've been making ctrl and meta equivalent for all keyboard shortcuts to make switching from mac to windows simpler